workbench-core/base/src/utilities/validatorHelper.ts

```diff
@@ -27,6 +37,22 @@ z.ZodString.prototype.swbId = function (prefix: string): ZodString {
   return this.regex(uuidWithLowercasePrefixRegExp(prefix), { message: 'Invalid ID' });
 };
 
+z.ZodString.prototype.nonHTML = function (): ZodString {
+  return this.regex(nonHtmlRegExp(), { message: 'Input contains HTML' });
```

Comment:
could we specify invalid characters in here
`Input cannot contain characters < > { } `